src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -346,6 +364,17 @@ bool PerformInteractionRequest::ProcessVRResponse(
     return false;
   }
 
+  if (first_responser_ == FirstPerformInteractionResponser::VR) {
+    // After VR.PerformInteraction response HMI should close UI popup window
+    // if UI.PerformInteraction response comes
+    // after VR.PerformInteraction response.
+    // In this case SDL should send UI_ClosePopUp request.
+    smart_objects::SmartObject hmi_request_params =
+        smart_objects::SmartObject(smart_objects::SmartType_Map);
+    hmi_request_params[hmi_request::method_name] = "UI.PerformInteraction";
```

Comment:
@YarikMamykin I think will be better to create const variable for the `UI.PerformInteraction` string.

Reply:
@IGapchuk fixed in https://github.com/smartdevicelink/sdl_core/pull/3122/commits/0190836450ab1d7e130bbb8e32315eb3325a9a68